seurat-filter.R

```diff
@@ -52,6 +52,13 @@ option_list = list(
     default = NA,
     type = 'character',
     help = "File name in which to store serialized R object of type 'Seurat'.'"
+  ),
+  make_option(
+    c("--min-cells"),
+    action = "store",
+    default = 0,
+    type = 'integer',
+    help = "Include only genes with detected expression in at least this many cells."
```

Comment:
I guess it would take us to the philosophical discussion of how much we want to "improve" packages through the wrappers, as opposed to simply expose their functionality and then pick the best one for each task by the different tools once they are interoperable.

Reply:
If we're being faithful to the underlying methods we should just shoe-horn this into the object creation as Seurat does :-P. But if we're being keen on consistency with the other tools (and existing arguments for cell-wise filtering) as you were saying the other day - which I think is a good plan - then I think we can allow ourselves some lattitude to 'tidy up' :-)

Reply:
I think that being keen on consistency shouldn't mean stretching the packages to do things that they don't have implementations for. In that case one would rather do a PR to the upstream package with the additional functionality. We should aim to be consistent in the sense of trying to put forward equivalent interfaces for each major step on each major tool, but only within the existing abilities of each tool.